doc/users/next_whats_new/fix_dash_offset_Patch.rst

```diff
@@ -0,0 +1,5 @@
+Fix the dash offset of the Patch class
+--------------------------------------
+Traditionally, when setting the linestyle on a patch object using a dash tuple the
```

Comment:
```suggestion
Traditionally, when setting the linestyle on a `.Patch` object using a dash tuple the
```
It's always good for navigation if we can cross-link the first mention of a code object.